source/views/components/filter/types.js

```diff
@@ -6,7 +6,8 @@ export type ToggleSpecType = {
 }
 
 export type ListItemSpecType = {|
-	title: string,
+	title: string | number,
+	label?: string,
```

Comment:
I'm not entirely clear on the difference between these two – can you summarize for me, please?

Reply:
Looks like it comes down to:
```js
title: term.term,
label: parseTerm(term.term.toString()),
```
* `term` would be numeric
* `parseTerm` lives in [lib/course-search/parse-term.js](https://github.com/StoDevX/AAO-React-Native/blob/master/source/lib/course-search/parse-term.js) where it returns the human readable term name (Abroad current / next year, Fall current / next year, etc)

Reply:
Yep! I changed the spec type so that the applyFilters() function would work when the value of the key item specified isn't a string. The label allows the filter options to be displayed as "Fall 2016/17" instead of 20171, for example.